docs/internal-documentation/maintenance/Garbage-Collection-Testing.md

```diff
@@ -0,0 +1,128 @@
+# Testing instructions for the Garbage Collection feature of the Zonemaster Backend module
+
+## Introduction
+The purpose of this instruction is to serve as a notice for manual testing of the new garbage collection feature at release time.
+
+## Testing the unfinished tests garbage collection feature
+
+1. Ensure that the database has the required additionnal columns for this feature:
+     ```
```

Comment:
This is not necessary anymore since we will test an installation based on updated code. Right?

Reply:
Is it really critical to not test this? It takes 5 second. This column is definitely necesseary for this testing procedure to make any sense even if it should be there now by default.